src/components/inputs/UrlInput.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import StringInput from './StringInput'
+import SmallError from '../util/SmallError'
+
+
+function validate (url) {
+  let error;
+  const getProtocol = (url) => {
+    try {
+      const urlObj = new URL(url);
+      return urlObj.protocol;
+    }
+    catch (err) {
+      return undefined;
+    }
+  };
+  const protocol = getProtocol(url);
+  if (
+    protocol &&
+    protocol === "http:" &&
+    window.location.protocol === "https:"
+  ) {
+    error = (
+      <SmallError>
+        CORs policy won&apos;t allow fetching resources served over http from https, use a <code>https://</code> domain
```

Comment:
```suggestion
        CORS policy won&apos;t allow fetching resources served over http from https, use a <code>https://</code> domain
```

Or is the lower case `s` intentional?

Reply:
Nope you're completely right